### ISSUE TITLE
[false-negative] Unnecessary parentheses `(!empty($var))`

### ISSUE THREAD

**Issue by ComiR** [kalessil/phpinspectionsea#649]
```php
if(
    // this gets detected
    (empty($var))
) {
    // do sth.
}
if(
    // this gets detected
    !(empty($var))
    /// this does not
    || (!empty($var)) 
) {
    // do sth. else
}
```
This is due to the additional unary operator `!`. 
It seems as if all operations in the parentheses break the detection (i.e. `($var === '')` doesn't get detected either). But the latter may be favored by some code style, so maybe we need to introduce some settings here?

**Comment by kalessil** [comment]
I'd prefer to not introduce settings. Will add `(!isset|empty(...))` constructs reporting - minimal efforts are needed.
